main.js

```diff
@@ -0,0 +1,32 @@
+var direction = {north: false, east: true, south: false, west: false};
```

Comment:
Instead of having a property for each direction, you should have one property named something like `direction` which holds a string value describing the direction. You can also use that string value to help you determine the class to apply to the car when the key is pressed, potentially 🤔